maquetado/login.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Log In</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="login-container">
+    <form class="info-container column">
+      <img src ="./assets/wolox.png" class="login-img" alt="Wolox">
+      <h3 class="input-description">Email</h3>
+      <input name="email" id="email" class="input-field">
+      <h3 class="input-description">Password</h3>
+      <input name="password" id="password" class="input-field">
+      <button href="tbdef" class="login-detail">
+        <h3 class="login-detail-font">Log in</h3>
+      </button>
+      <button href="tbdef" class="sign-up-detail">
+        <p class="sign-up-detail-font">Sign Up</p>
```

Comment:
There is not need to wrap this inside a p tag. You can write this inside your button without using a tag as container.